Modules/expat/expat_external.h

```diff
@@ -7,6 +7,10 @@
 
 /* External API definitions */
 
+/* Namespace external symbols to allow multiple libexpat version to
+   co-exist. */
+#include "pyexpatns.h"
```

Comment:
I would be nice to upstream pyexpatns.h namespace, it seems useful to build a clean DLL on Windows.